docs/developer/extending/plugins.mdx

```diff
@@ -109,14 +104,24 @@ There is no need to implement all base methods as the `PluginsManager` will use
 
 ### Loading your plugin
 
-To activate the new plugin, add it to the `settings.PLUGINS` list in your Django settings:
+Saleor uses the *entry points* API of Python's `setuptools` to automatically discover installed plugins. To make a plugin discoverable, include a `saleor.plugins` entry point in your `setup()` call. The syntax is `package_name = package_name.path.to:PluginClass`:
 
 ```python
-# settings.py
-
-PLUGINS = ["saleor.plugins.custom.plugin.CustomPlugin", ]
```

Comment:
This approach still works but I understand it's only because we have our plugins in Saleor repo and don't recommend that to others building their own plugins?

Reply:
It works, in fact discovered plugins are appended to this list. We don't want to recommend modifying core's source code to install a plugin.